### PR TITLE
Details: Allowing copy paste using TextInput on both devices

### PR DESCRIPTION
**Pull request by jwallet**

### CODE REVIEW

**Comment by alexbrazier** [comment]
Thanks @jwallet, testing this on an ios simulator with the existing code there already seems to be copy/paste functionality. Does setting it to `editable` fix it on android or is there something else I'm missing?

image

Setting it to `editable` seems to allow pasting and briefly typing into the text input before it gets reset, so wonder if it's best to keep it as `editable:false` on ios assuming the fix is for android only?

**Comment by jwallet** [comment]
Yes my bad I opened this too early. But on android editable blocks the copy paste functionality. Give me some times to work on it...

**Comment by jwallet** [comment]
my bad, ''editable:false'' actually works on android

File: src/components/RequestDetails.tsx [hunk 1]
```diff
@@ -48,30 +48,19 @@ const Headers = ({
...
-  }
-
+  // always use TextInput for large text on both devices instead of Text:
+  // - on iOS, Text has a limit, not for TextInput
+  // - on Android, copy paste is not allowed on a non-editable TextInput
   return (
-    <View style={styles.largeContent}>
-      <ScrollView nestedScrollEnabled>
-        <View>
-          <Text style={styles.content}>{children}</Text>
```
[the remark this answers is not in the document]
Comment (jwallet) [reply src/components/RequestDetails.tsx:71]: Oh I did not think about looking for a prop like that, you can close it then.

**Comment by alexbrazier** [comment]
Just opened #82 which should cover it

**Pull request closed without merging**